jina/parsers/orchestrate/pod.py

```diff
@@ -88,16 +88,24 @@ def mixin_pod_parser(parser, pod_type: str = 'worker'):
         else argparse.SUPPRESS,
     )
 
-    gp.add_argument(
-        '--port',
-        '--port-in',
-        type=str,
-        default=str(helper.random_port()),
-        help='The port for input data to bind to, default is a random port between [49152, 65535].'
-        'In the case of an external Executor (`--external` or `external=True`) this can be a list of ports, '
-             'separated by commas. '
-        ' Then, every resulting address will be considered as one replica of the Executor.',
-    )
+    port_description = (
+        'The port for input data to bind to, default is a random port between [49152, 65535]. '
+        'In the case of an external Executor (`--external` or `external=True`) this can be a list of ports, separated by commas. '
+        'Then, every resulting address will be considered as one replica of the Executor.'
+    )
+
+    if pod_type != 'gateway':
+        gp.add_argument(
+            '--port', '--port-in', type=str, default=str(helper.random_port()), help=port_description
+        )
+    else:
+        gp.add_argument(
```

Comment:
can u show me a CLI example of how does this work?

Reply:
jina gateway --port 12345 12345
or
jina gateway --port 12345

Reply:
why didn't we do like this in the past for other cases?

Reply:
not sure, probably we were just fine with the comma separated list of values format